config/config.txt

```diff
@@ -543,3 +543,9 @@ DEFAULT_VIEW_SQUARE 15x15
 
 ## Add the ID of the role you want assigning here
 #DISCORD_ROLEID 000000000000000000
+
+## How long in seconds after which a hard delete is treated as causing lag. This can be a float and supports a precision as low as nanoseconds.
+#HARD_DELETES_OVERRUN_THRESHOLD 0.5
+
+## Once a typepath causes overrun from hard deletes this many times, stop hard deleting it on garbage collection failures. (set to 0 to disable)
+#HARD_DELETES_OVERRUN_LIMIT 0
```

Comment:
the threshold needs to be enabled for the overrun limit to do anything doesn't it?

Reply:
yep. if threshold is set to 0, even the admin notices get disabled, and the if check for that overrun limit will never run.

the postpone code will still run.

Reply:
It's a little unclear just from the config option, tho maybe that's better then giving people who won't read the code an understanding of it

Reply:
its 0.5 by default in code, hence the comment out in the config